Gemfile

```diff
@@ -2,15 +2,19 @@ source 'https://rubygems.org'
 
 gemspec
 gem 'fog-aws'
-gem 'capybara', '~> 2.3.0'
-gem 'database_cleaner', '~> 0.9.1'
-gem 'rspec-rails', '~> 3.2.0'
-gem 'poltergeist'
 gem 'ammeter'
-gem 'test-unit', platforms: [:ruby_22]
 
-group :development do
+group :development, :test do
```

Comment:
The removal of the `:test group was deliberate - happened in https://github.com/withassociates/slices/commit/d22e903db0ea96f63df80fd58038509033c0fe49. How come you added it back? Was it necessary for upgrading to rails 4 (it was in that commit)?

I think the reasoning at the time was that as Slices is a gem and not an app it never runs in a "production" mode - the gems in here are for testing and development only. Development gems are superset of test gems, so made sense to make "test" be the _default_ group.